src/firmware/app/world/firmware_world.h

```diff
@@ -24,20 +24,22 @@ FirmwareWorld_t* app_firmware_world_create(FirmwareRobot_t* robot, FirmwareBall_
  * NOTE: This will not destroy the values pointed to by any pointers passed to the
  *       `create` function
  *
- * @param world The world to destroy
+ * @param world [in] The world to destroy
```

Comment:
these are pseudoclasses, so don't need to change

Reply:
Done.